src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -168,10 +168,27 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
 
     const pairedAddress = pairAssetAddress ?? ''
 
+    const asymDestinationAssetId = (() => {
+      if (isRuneTx && opportunityType !== 'sym') return thorchainAssetId
+      if (!isRuneTx && opportunityType !== 'sym') return assetId
```

Comment:
[question]: Its possible for this iife to return `undefined`. Is is intended noting `assetIdToPoolAssetId` expects a defined `assetId`?